python/plugins/db_manager/layer_preview.py

```diff
@@ -78,11 +78,17 @@ def setDirty(self, val=True):
     def _clear(self):
         """ remove any layers from preview canvas """
         if self.item is not None:
-            ## skip exception on RuntimeError fixes #6892
             try:
                 self.item.aboutToChange.disconnect(self.setDirty)
-            except RuntimeError:
-                pass
+            ## skip exception on RuntimeError fixes #6892
+            except RuntimeError as ex:
+                self.parent.infoBar.pushMessage(str(ex), QgsMessageBar.WARNING,
```

Comment:
Maybe better to use unicode here, to avoid possible errors with non-ASCII characters